webpack.config.js

```diff
@@ -1,16 +1,27 @@
-'use strict';
-
 var path = require('path');
 var webpack = require('webpack');
 var autoprefixer = require('autoprefixer');
 
 /**
  * # Webpack Build Process for Data Lasso
  *
- * Run with `--production` to build minified production ready distributive
+ * Available arguments:
+ *
+ * -- production
```

Comment:
Remove the extra space? So, `--production`. Same with `--watch` and `--minified`